templates/servicemonitor.yaml

```
@@ -1,28 +1,30 @@
-{{- if .Values.serviceMonitor.enabled -}}
+{{- range $name, $values := .Values.Servers }}
+{{- if $values.serviceMonitor.enabled -}}
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
-  name: {{ include "<CHARTNAME>.fullname" . }}-sm
-  namespace: {{ $.Values.serviceMonitor.namespace }}
+  name: {{ include "<CHARTNAME>.fullname" $ }}-{{ $name }}-sm
```

`-sm` suffix is redundant
```suggestion
  name: {{ include "<CHARTNAME>.fullname" $ }}-{{ $name }}
```